tests/Integration/Http/Fixtures/PostResourceWithJsonOptions.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Illuminate\Tests\Integration\Http\Fixtures;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class PostResourceWithJsonOptions extends JsonResource
+{
+    public function toArray($request)
+    {
+        return [
+            'id' => $this->id,
+            'title' => $this->title,
+            'reading_time' => $this->reading_time,
+        ];
+    }
+
+    public function jsonOptions()
+    {
+        return JSON_PRESERVE_ZERO_FRACTION;
```

Comment:
This seeems to be bound to only one implementation at a time. What if I want to use it with something like `JSON_UNESCAPED_SLASHES `. How do I go about that?

Reply:
The options are integers used as bit masks. If you want multiple options, go for `JSON_PRESERVE_ZERO_FRACTION|JSON_UNESCAPED_SLASHES`

Reply:
Take a look at https://www.php.net/manual/en/function.json-encode.php ;-)